### DISCUSSION TITLE
[Config Support]: Disabling all logging other then errors

### DISCUSSION THREAD

**Discussion by leonbrag** [blakeblackshear/frigate#12839] [Config Support] [answered]
### Describe the problem you are having

I am trying to disable all the log output other then errors.  However, despite all different ways I try, there are lots of log messages.

Example of messages, which seem all come from Nginx:

![image](https://github.com/user-attachments/assets/108210f1-3863-4ad0-8255-cdb7ff2076d9)

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
logger:
  # Optional: Default log verbosity (default: shown below)
  default: error
  logs:
    frigate: error
    go2rtc: error
    nginx: error
    Nginx: error
```

### Relevant log output

```shell
Example of message I am trying to disable:

frigate  | 2024-08-08 12:03:32.738603079  172.18.0.1 - - [08/Aug/2024:12:03:32 -0700] "GET /api/Garage/latest.jpg?h=288 HTTP/1.1" 200 19888 "http://hpdebian:8123/api/hassio_ingress/jX6C7VDVH8WP3GjDkjqg1tscCj-igJJWWj_qnVAuozk//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0" "192.168.4.204, 172.30.32.1"
frigate  | 2024-08-08 12:03:32.747816286  172.18.0.1 - - [08/Aug/2024:12:03:32 -0700] "GET /api/Side_Yard/latest.jpg?h=288 HTTP/1.1" 200 23072 "http://hpdebian:8123/api/hassio_ingress/jX6C7VDVH8WP3GjDkjqg1tscCj-igJJWWj_qnVAuozk//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0" "192.168.4.204, 172.30.32.1"
frigate  | 2024-08-08 12:03:34.408785500  172.18.0.1 - - [08/Aug/2024:12:03:34 -0700] "GET /api/go2rtc/api HTTP/1.1" 200 138 "http://hpdebian:8123/system" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0" "192.168.4.204, 172.30.32.1"
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

OpenVino

**Comment by NickM-27** [comment, marked as answer]
Nginx logs can't be disabled from the config. You'd need to disable in the nginx.conf

**Comment by stigglitch** [comment]
Run these two lines inside the container, tested with 0.16, your system might explode, NO WARRANTIES!

The 1st one is for nginx system events., the second is for client events. It survives reboots.

sed -i 's#error_log /dev/stdout warn;#error_log /dev/stdout error;#' /usr/local/nginx/conf/nginx.conf
sed -i 's#access_log /dev/stdout main;#access_log off;#' /usr/local/nginx/conf/nginx.conf
